include/ebpf_structs.h

```diff
@@ -16,6 +16,8 @@ typedef enum bpf_map_type
     BPF_MAP_TYPE_ARRAY = 2,       ///< Array, where the map key is the array index.
     BPF_MAP_TYPE_PROG_ARRAY =
         3, ///< Array of program fds usable with bpf_tail_call, where the map key is the array index.
+    BPF_MAP_TYPE_PERCPU_HASH = 4,
+    BPF_MAP_TYPE_PERCPU_ARRAY = 5,
```

Comment:
You also need to add these into libs\api_common\windows_platform_common.cpp windows_map_types() array around line 113 so verification will work